src/plots/gl3d/scene.js

```diff
@@ -243,7 +244,26 @@ function tryCreatePlot(scene, cameraObject, pixelRatio, canvas, gl) {
         }
     }
 
-    return failed < 2;
+    if(failed < 2) {
+        scene.wheelListener = mouseWheel(scene.glplot.canvas, function(dx, dy) {
```

Comment:
A couple questions:

- why can't we add logic to this block:

https://github.com/plotly/plotly.js/blob/4a649f01f21fbf3ec814242a0b1f93ca9d2f8067/src/plots/gl3d/scene.js#L296-L300

which looks like also adds a mouse-wheel handler to the scene

- Shouldn't we also emit a `plotly_relayout` event with the `aspectratio` keys just like we do here:

https://github.com/plotly/plotly.js/blob/4a649f01f21fbf3ec814242a0b1f93ca9d2f8067/src/plots/gl3d/scene.js#L283-L290

for perspective projections?

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/4292/commits/fe19e47430efc9ebab484ce5113bd7841b62a5cd